Form-Controls/results.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
```

Comment:
Can you talk through this file?

Reply:
I've deleted the page , we don't need that page.